adit_radis_shared/accounts/tests/test_views.py

```
@@ -0,0 +1,82 @@
+    assert "Group B" in payload["toast"]["text"]
+
+
+@pytest.mark.django_db
+def test_active_group_invalid_id_raises_validation_error(client: Client):
+    user = UserFactory.create()
+    group = GroupFactory.create()
+    add_user_to_group(user, group)
+    client.force_login(user)
+
+    # A non-integer group id triggers a ValidationError inside the view.
+    with pytest.raises(Exception):
```

Ruff (0.15.20)

[warning] Do not assert blind exception: `Exception`

(B017)